distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1204,6 +1204,92 @@ public void testNoEnforceValidityErrorsOrWarnings() throws Exception {
     }
 
     @Test
+    public void testQueryErrors() throws Exception {
+        getServiceManager().startFeature(true,
+                "catalog-plugin-metacard-validation",
```

Comment:
metacard validation is started by default now, you shouldn't need to start it explicitly anymore